src/main/resources/public/scss/projectSettings/tabViews/autoAnalysis/autoAnalysis.scss

```diff
@@ -0,0 +1,290 @@
+.auto-analysis-project-settings{
+  padding: 10px 0 20px;
+  position: relative;
+  .mobile-disabling-cover {
+    @extend %MOBILE_DISABLING_COVER;
+  }
+  .mode-input{
+    max-width: 130px;
+  }
+  .in-progress{
+    display: block;
+    margin-left: 30px;
+    color:  $COLOR--tealish;
+    font-size: 13px;
+  }
+  .aa-base{
+    height: 50px;
+    flex-direction: column;
+    display: flex;
```

Comment:
If we change browsers support only with 5.0 version (not right now), rewrite styles without using of flexbox.